sceptre/cli/drift.py

```diff
@@ -0,0 +1,108 @@
+import click
+from click import Context
+
+from sceptre.context import SceptreContext
+from sceptre.cli.helpers import (
+    catch_exceptions,
+    write
+)
+from sceptre.plan.plan import SceptrePlan
+
+
+BAD_STATUSES = [
+    "DETECTION_FAILED",
+    "TIMED_OUT"
+]
+
+
+@click.group(name="drift")
+def drift_group():
+    """
+    Commands for calling drift detection.
+    """
+    pass
+
+
+@drift_group.command(name="detect", short_help="Run detect stack drift on running stacks.")
+@click.argument("path")
+@click.pass_context
+@catch_exceptions
+def drift_detect(ctx: Context, path: str):
+    """
+    Detect stack drift and return stack drift status.
+
+    In the event that the stack does not exist, we return
+    a DetectionStatus and StackDriftStatus of STACK_DOES_NOT_EXIST.
+
+    In the event that drift detection times out, we return
+    a DetectionStatus and StackDriftStatus of TIMED_OUT.
+
+    The timeout is set at 5 minutes, a value that cannot be configured.
+
+    :param path: The path to execute the command on.
+    :type path: str
+    """
+    context = SceptreContext(
+        command_path=path,
+        project_path=ctx.obj.get("project_path"),
+        user_variables=ctx.obj.get("user_variables"),
+        options=ctx.obj.get("options"),
+        output_format=ctx.obj.get("output_format"),
+        ignore_dependencies=ctx.obj.get("ignore_dependencies")
+    )
+
+    plan = SceptrePlan(context)
+    responses = plan.drift_detect()
+
+    exit_status = 0
+    for stack, response in responses.items():
+        status = response["DetectionStatus"]
+        if status in BAD_STATUSES:
+            exit_status += 1
+        for key in ["Timestamp", "ResponseMetadata"]:
+            response.pop(key, None)
+        write({stack.external_name: response}, context.output_format)
```

Comment:
This seems to ignore the sceptre `--output [text|yaml|json]` option.  It always outputs in json.  is there a reason for this?

Reply:
Not quite true. In `output=text` mode it emits a Python dict. The issue I assume you ran into is that in `output=yaml` it continues to emit JSON due to [this](https://github.com/Sceptre/sceptre/issues/1173) bug that I raised.

Reply:
ok, i see that output flag is a bit messed up.